src/main/scala/devices/debug/SBA.scala

```diff
@@ -177,17 +177,17 @@ object SystemBusAccessModule
                                        Mux((tryWrEn || tryRdEn) && sbAlignmentError, 3.U, SBCSFieldsReg.sberror))))) // Address alignment error
       SBCSFieldsReg.sbaccess        := Mux(SBCSWrEn, SBCSWrData.sbaccess, SBCSFieldsReg.sbaccess)
       SBCSFieldsReg.sbversion       := 1.U(1.W) // This code implements a version of the spec after January 1, 2018
-      SBCSFieldsReg.sbbusy          := (sb2tl.module.io.sbStateOut =/= SystemBusAccessState.Idle.id.U)
-      SBCSFieldsReg.sbasize         := sb2tl.module.edge.bundle.addressBits.U
-      SBCSFieldsReg.sbaccess128     := (cfg.maxSupportedSBAccess == 128).B
-      SBCSFieldsReg.sbaccess64      := (cfg.maxSupportedSBAccess >=  64).B
-      SBCSFieldsReg.sbaccess32      := (cfg.maxSupportedSBAccess >=  32).B
-      SBCSFieldsReg.sbaccess16      := (cfg.maxSupportedSBAccess >=  16).B
-      SBCSFieldsReg.sbaccess8       := (cfg.maxSupportedSBAccess >=   8).B
+      SBCSFieldsReg.sbbusy          := sbBusy
```

Comment:
I don't really have a problem with this change (not enough to block the PR), but do you ever need SBCSFieldsReg.sbbusy anymore? Couldnt you just use sbBusy everwhere?

If you don't use it Chisel will remove it anyway, but just curious.